media-proxy/src/mesh/conn_rdma.cc

```diff
@@ -0,0 +1,399 @@
+#include <queue>
+#include "conn_rdma.h"
+
+namespace mesh::connection {
+
+std::atomic<int> Rdma::active_connections(0);
+
+Rdma::Rdma() : ep_ctx(nullptr), init(false), trx_sz(0), mDevHandle(nullptr) {
+    std::memset(&ep_cfg, 0, sizeof(ep_cfg));
+    ++active_connections;
+}
+
+Rdma::~Rdma() {
+    on_shutdown(context::Background());
```

Comment:
on_shutdown return a return value. It is not checked here and in other places.

Don't we have any warnings enabled that would inform us about things like that? something like `-Wunused-result`. Could you possibly enable those warnings? In my opinion we should enable all warnings and threat them as errors.

Reply:
@moleksy Please include this in future refactoring.